src/matrixfree/initialConditions.cc

```diff
@@ -296,6 +296,71 @@ MatrixFreePDE<dim, degree>::applyInitialConditions()
         }
     }
 
+  // 1) DETERMINE UNIQUE VTK FILES, CREATE DICTIONARY
+  std::unordered_map<std::string, std::vector<size_t>> file_field_map;
+  for (size_t i = 0; i < userInputs.number_of_variables; ++i) {
+      if (userInputs.load_ICs[i]){
+          file_field_map[userInputs.load_file_name[i]].push_back(i);
+      }
+  }
+  // Read out unique filenames
+  int proc_num = Utilities::MPI::this_mpi_process(MPI_COMM_WORLD);
+  if (proc_num == 0){
+      std::cout << "Unique VTK input files: " << std::endl;
```

Comment:
Does this not output "unique VTK inputs files: " even when the user is using the initial conditions file (`ICs_and_BCs.cc`).

To prevent confusion, a conditional should be added so we only print when load vtk is specified.